test/unit/remote-config/remote-config.spec.ts

```diff
@@ -51,6 +51,7 @@ describe('RemoteConfig', () => {
             'android_en': { value: 'A Droid must love a pumpkin spice latte.' },
           },
           description: 'Description of the parameter.',
+          valueType: 'STRING' as remoteConfig.ParameterValueType,
```

Comment:
Where is this used?

Reply:
This is used in `REMOTE_CONFIG_RESPONSE` in line 98 and also be used in the parameter groups tests (ex: line 506)